parts/k8s/cloud-init/artifacts/setup-custom-search-domains.sh

```diff
@@ -2,7 +2,7 @@
 set -x
 source /opt/azure/containers/provision_source.sh
 
-sudo echo "  dns-search <searchDomainName>" >> /etc/network/interfaces.d/50-cloud-init.cfg
```

Comment:
I think this script runs as root, so perhaps this wasn't causing an actual bug.

Reply:
yeah, let's enable the check but also get rid of the `sudo`